docs/options.md

```diff
@@ -177,7 +177,9 @@ A shell command to run before building the wheel. This option allows you to run
 
 If dependencies are required to build your wheel (for example if you include a header from a Python module), set this to `pip install .`, and the dependencies will be installed automatically by pip. However, this means your package will be built twice - if your package takes a long time to build, you might wish to manually list the dependencies here instead.
 
-The active Python binary can be accessed using `python`, and pip with `pip`; `cibuildwheel` makes sure the right version of Python and pip will be executed. `{project}` can be used as a placeholder for the absolute path to the project's root.
+The active Python binary can be accessed using `python`, and pip with `pip`; `cibuildwheel` makes sure the right version of Python and pip will be executed. `{project}` can be used as a placeholder for the absolute path to the project's root and will be replaced by `cibuildwheel`.
```

Comment:
What has changed here?

Reply:
I've added ` and will be replaced by `cibuildwheel`` to the end of the line, like it was already specified for `CIBW_TEST_COMMAND`.

Reply:
Aha, that's why :-)